src/player.ts

```diff
@@ -1716,6 +1701,46 @@ export default class Player extends FakeEventTarget {
     return this._engine.getDrmInfo();
   }
 
+  private _getSubtitleStyleSheet(): CSSStyleSheet {
+    let element = Utils.Dom.getElementBySelector(`.${this._playerId}.${SUBTITLES_STYLE_CLASS_NAME}`);
+    if (!element) {
+      element = Utils.Dom.createElement('style');
+      Utils.Dom.addClassName(element, this._playerId);
+      Utils.Dom.addClassName(element, SUBTITLES_STYLE_CLASS_NAME);
+      Utils.Dom.appendChild(document.head, element);
+    }
+    return element.sheet;
+  }
+
+  private _resetCustomSubtitleStyles(): void {
+    const sheet = this._getSubtitleStyleSheet();
+    while (sheet.cssRules.length) {
+      sheet.deleteRule(0);
```

Comment:
is this better than removing the sheet and adding it again ?

or than using replace() ?

Reply:
`this._getSubtitleStyleSheet` returns CSSStyleSheet (creates if the CSSStyleSheet absent).
Removing add complexity, since we would need 2 methods - one for reset (that removes then re-creates) and another one for get existed CSSStyleSheet